src/pages/[userId]/pledges.js

```diff
@@ -1,7 +1,46 @@
 import { useRouter } from 'next/router';
 
+import Grid from '../../utils/Grid';
+
+import Pledge from '../../components/Pledge';
+
 export default function Pledges() {
   const router = useRouter();
   const { userId } = router.query;
-  return <p>Pledges for {userId}</p>;
+
+  const pledgesArr = [
+    {
+      id: 1,
+      first_name: 'Jon',
+      last_name: 'Smith',
+    },
+    {
+      id: 2,
+      first_name: 'Matt',
+      last_name: 'Clark',
+    },
+    {
+      id: 3,
+      first_name: 'Mitchell',
+      last_name: 'Pavel',
+    },
+    {
+      id: 4,
+      first_name: 'Bob',
+      last_name: 'Chin',
+    },
+  ];
+
+  const pledges = pledgesArr.map((pledge) => <Pledge key={pledge.id} donor={pledge} />);
```

Comment:
Since this is so concise already, we may just want to put it right into the markup. It may get a little more complicated later though when we start adding more functionality with the pledges. We can just worry about it later, but I wanted at least make a note about it.

Also after thinking about it more I think we should use the prop name `pledges` here, and do the same on the other pages with their respective names. In addition we are going to need to change our language around donors. Dani was telling me that FOCUS is trying to use the term "ministry partner" instead. The unfortunate thing is it's so many more characters lol. But it's helpful to have our code reflect the actual names so it will be worth making the change.